lib/foreman_remote_execution/engine.rb

```diff
@@ -20,7 +20,7 @@ class Engine < ::Rails::Engine
     initializer 'foreman_remote_execution.configure_assets', :group => :assets do
       SETTINGS[:foreman_remote_execution] =
         {:assets => {:precompile => [
-          'template_input.js',
```

Comment:
I don't like this cop. Why it's harmful to keep comma after last item? I like to use it.